homeassistant/components/mqtt/update.py

```diff
@@ -99,10 +109,22 @@ def __init__(
         """Initialize the MQTT update."""
         self._config = config
         self._attr_device_class = self._config.get(CONF_DEVICE_CLASS)
+        self._attr_release_summary = self._config.get(CONF_RELEASE_SUMMARY)
+        self._attr_release_url = self._config.get(CONF_RELEASE_URL)
+        self._attr_title = self._config.get(CONF_TITLE)
+        self._entity_picture: str | None = self._config.get(CONF_ENTITY_PICTURE)
```

Comment:
I suggest that you add type hints right away.
Are they all `str | None` ?

Reply:
Do you mean adding typing for `_attr_*` variables? They have types defined in the `UpdateEntity` class.

Reply:
doh sorry!!!

Reply:
no worries